x-pack/plugins/security_solution/server/lib/detection_engine/routes/signals/set_alert_tags_route.test.ts

```diff
@@ -65,17 +67,40 @@ describe('setAlertTagsRoute', () => {
         status_code: 400,
       });
     });
+
+    test('returns 400 if no alert ids are provided', async () => {
+      request = requestMock.create({
+        method: 'post',
+        path: DETECTION_ENGINE_ALERT_TAGS_URL,
+        body: getSetAlertTagsRequestMock(['tag-1'], ['tag-2']),
+      });
+
+      context.core.elasticsearch.client.asCurrentUser.updateByQuery.mockResponse(
+        getSuccessfulSignalUpdateResponse()
+      );
+
+      const response = await server.inject(request, requestContextMock.convertContext(context));
+
+      context.core.elasticsearch.client.asCurrentUser.updateByQuery.mockRejectedValue(
+        new Error('Test error')
+      );
+
+      expect(response.body).toEqual({
+        message: [`No alert ids were provided`],
+        status_code: 400,
+      });
+    });
   });
 
   describe('500s', () => {
     test('returns 500 if ', async () => {
```

Comment:
Do we need to finish this test description?

Reply:
Good catch, updating now